src/js/components/productSupplier/create/subsections/DefaultPreferenceType.jsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+import { Controller } from 'react-hook-form';
+
+import DateField from 'components/form-elements/v2/DateField';
+import SelectField from 'components/form-elements/v2/SelectField';
+import TextInput from 'components/form-elements/v2/TextInput';
+import Subsection from 'components/Layout/v2/Subsection';
+import { useSelector } from 'react-redux';
+
+const DefaultPreferenceType = ({ control, errors }) => {
+  const {
+    preferenceTypes,
+  } = useSelector((state) => ({
+    preferenceTypes: state.productSupplier.preferenceTypes,
+  }));
+
+  return (
+    <Subsection
+      collapsable={false}
+      title={{
+        label: 'react.productSupplier.subsection.defaultPreferenceType.title',
+        defaultMessage: 'Default Preference Type',
+      }}
+    >
+      <div className="row">
+        <div className="col-lg col-md-6">
+          <Controller
+            name="defaultPreferenceType"
+            control={control}
+            render={({ field }) => (
+              <SelectField
+                {...field}
+                placeholder="Select"
+                title={{
+                  id: 'react.productSupplier.form.defaultPreferenceType.title',
+                  defaultMessage: 'Default Preference Type',
+                }}
+                tooltip={{
+                  id: 'react.productSupplier.form.defaultPreferenceType.tooltip',
+                  defaultMessage: 'Company-wide purchasing preference for this supplier established through a competitive bid',
+                }}
+                options={preferenceTypes}
+                errorMessage={errors.defaultPreferenceType?.message}
```

Comment:
One change related to the requested changes in my previous PR is missing. To display the error you need `hasErrors={true}` (only in select fields)